Makefile

```
@@ -33,15 +33,36 @@ endif
 #########################################
 # Rules for local development scenarios #
 #########################################
+.PHONY: downlaod-kubeconfig
+download-kubeconfig:
+	@echo "enter project name"; \
+	read PROJECT; \
+	echo "enter seed name"; \
+	read SEED; \
+	echo "enter shoot name"; \
+	read SHOOT; \
+	bash ./hack/local_setup.sh --SEED $$SEED --SHOOT $$SHOOT --PROJECT $$PROJECT;
```

see if we can run the script without bash as suggested below with `@` maybe
```suggestion
	./hack/local_setup.sh --SEED $$SEED --SHOOT $$SHOOT --PROJECT $$PROJECT;
```

---

it doesn't require anything just accepting the suggestion.